### ISSUE TITLE
remove additional data from processed files

### ISSUE THREAD

**Issue by christinabergmann** [langcog/mb1-analysis#154]
The following files need to have columns removed:
- CogDevLab-BYU Participants Data.xlsx (birth date and test date)
- labunam_Participants_* (all files, birth date and test date)
- lscp-psl_manybabies_Participants.csv (test date and names)
- dcnlab-tennessee participants.xlsx (birth date and test date, also sibling birth dates)
- isl_notredame_Participants Data.csv (birth date and test date)
- babylab-lancaster* (all files, birth date and test date)

**Comment by christinabergmann** [comment]
update: labunam also has a "name" column that I deduce to not contain RA info (since it's different for every participant).

**Comment by rebeccalundwall** [comment]
@christinabergmann re labunam  The "id" column has codes like mb_08, mb_25 -- so my guess is that this is the column we keep.  The other numbers may be a continuous run of participants in this lab (some who participated in other studies).  I do not know why we do not have subjects mb_01 to mb_07 or mb-11p to mb_24.  Perhaps these are potential participants who never entered the lab.  Who wants to email the lab and ask?

Meanwhile, I forked the repo and and trying to remove the columns you suggested using GitHub desktop.  I'm having to experiment to see what works.  My understanding is that I clone to desktop, sync to my branch, and then do a pull request to have someone examine my edits.  If you know different or who I should talk to, please let me know.

**Comment by christinabergmann** [comment]
Yep, you're right about using github I think. It's great that you references the issue, I am sorry for making you do double work. I should have assigned myself. 

I agree in principle about id versus subid, but then we need to check that the trial data contains the same info and rename columns in both files, because the script right now only looks at the subid column. So it might be safer to not touch them (otherwise we might introduce mistakes). My bigger concern is that there is no RA info, this should be checked.

**Comment by christinabergmann** [comment]
(This is a separate point so I am making a new comment). 
Having a few more participants in the participant file has been taken to mean those participants entered the lab but were never tested, as far as I know. I remember checking these things before ICIS and focusing on fixing things when there were more participants in the trial data than vice versa and the participant file had a strict superset of participants (see also the programmatic fixes here: https://github.com/langcog/mb1-analysis/blob/master/01_read_and_merge.Rmd). I think others looked at this as well, but another set of eyes might not hurt, it's easy to miss things. (But don't feel obliged, this is a really, really awful job!)

**Comment by melsod** [comment]
In my quick glance it looks like the subids match up and there have been a number of issues with labunam files so far, so I think it's best to leave well enough alone if so. I think it's worth exploring if there's a mismatch between the participants and trials file, but probably not if there's "gaps" in the numbering scheme (there are a number of legitimate reasons that could happen, e.g. training a new RA on some practice data).

As for when there is a participant in the participants file but not in the trials file, that has generally been because the participant came into the lab but was too fussy to even complete a single trial (or for some other reason). We need to make sure those get logged in the appropriate discard category. Mike was suggesting adding a single trial to the trials file (see ), but I didn't want to implement that until we were sure that was the correct approach.
